DOCUMENTATION.md

```diff
@@ -0,0 +1,196 @@
+# Geocint Open Source documentation
+
+* Geocint folder structure
+* Geocint open-source installation and first run guide
+    * Installation
+    * First run
+* How geocint pipeline work
+    * How start_geocint.sh works
+    * How to write targets
+    * User schemas
+    * How to analyse build time for tables
+* Best practices of using
+
+## Geocint folder structure
+
+Geocint consists of 3 different parts:
+- [geocint-runner](https://github.com/konturio/geocint-runner) - a core part of the pipeline, includes utilities and initial Makefile
+- [geocint-openstreetmap](https://github.com/konturio/geocint-runner) - a chain of targets for downloading, updating and uploading
+to database OpenStreetMap planet dump
+- [geocint-private] any repository that contains your additional functionality
+
+During the deployment process, you should clone all these repos to ~/.
+Then start_geocint.sh will copy files from all these repositories to ~/geocint folder. Geocint folder will be the working folder for geocint pipeline
+
+In general case ~/geocint folder includes the next folders:
+- [functions/](functions) - service SQL functions, used in more than one other file
+- [procedures/](procedures) - service SQL procedures, used in more than one other file
+- [scripts/](scripts) - scripts that perform data transformation on top of table without creating new table
+- [tables/](tables) - SQL, which generates a table with the same name as the script
+- [static_data](static_data) - static file-based data stored in geocint repository
+All these folders are removed and recreated each time when geocint pipeline starts.
+
+After running the pipeline, make will create additional folders. These folders are used to store input (in folder), intermediate (mid folder), and output (out folder) data files.
+Make-profiler uses the timestamps of these files as timestamps for the execution of such a target:
+- [data/](data) - file-based input, middle, output data.
+	- data/in - all input data, downloaded elsewhere
```

Comment:
but planet.psm.pbf not in data/in it is in data

Reply:
this is how it was originally designed